test/index-spec.js

```diff
@@ -68,4 +67,41 @@ describe('Connection Test', () => {
         done();
       });
   });
+
+
+  it('sice -> rice', (done) => {
+    describe('Caller should send a ICE Candidate to server '
+      + 'and Server should broadcast the ICE Candidate', () => {
+      // given
+      const caller = io.connect(url, null);
+      const callee = io.connect(url, null);
+      const roomNumber = 12345;
+      caller.emit('join', roomNumber);
+      callee.emit('join', roomNumber);
+
+      const candidate = { room: roomNumber,
+        sdpMid: 'IamSDPMid',
+        sdpMLineIndex: '12345',
+        candidate: 'candidate:1234567890 1 udp 0987654321 192.168.0.1' };
+      const observable = Rx.Observable
+        .fromEvent(callee, 'rice')
+        .first();
```

Comment:
I know that `first()` returns `Single` object, But why this name is `observable`?

Reply:
In [RxJS Docs for 'first'](https://rxjs-dev.firebaseapp.com/api/operators/first), first() returns an Observable.
Meanwhile, because variable name ```observable``` is so general, I just edit ```observable``` to ```riceObservable```